doc/source/index.rst

```diff
@@ -12,8 +12,8 @@ Currently pyorbital only supports low earth orbit satellites.
 
 Installation
 ------------
-Pyorbital comes with a file platforms.txt that maps satellite name to NORAD identifier.
-This file needs to be copied to the appropriate satpy etc directory ($PPP_CONFIG_DIR).
+Pyorbital comes with a file platforms.txt that maps a satellite name to the NORAD identifier.
+This file needs to be copied to the appropriate config directory pointed to by :envvar:`PYORBITAL_CONFIG_PATH`.
```

Comment:
These sentences make it seem like I *need* to create this `platforms.txt`, but the envvar description makes it seem optional. If it is required then these sentences here need to mention `platforms.txt` by name.

Reply:
I tried improving this, thanks!